code/Dependencies.md

```diff
@@ -38,3 +38,34 @@ To add a new dependency evaluate the following things:
 * Check how much it affects `storj/uplink` size.
 
 Note: many of these are fixable with communication or making our own PR-s.
+
+## Finding the changed code in dependencies
+
+The easiest way to find the minimal set of code that needs to be reviewed is to:
+
+``` sh
+# create a temporary git branch
+git switch -c deps
+
+# vendor direct and indirect dependencies
+go mod vendor
+
+# commit the vendor directory
+git commit -a -m"before"`
+
+# add the new dependency
+go get example.test/new-dependency@latest
+modify-code to include the packages
+
+# update the vendor
+go mod vendor
+
+# commit the vendor directory
+git commit -a -m"after"`
+```
+
+The diff between those commits show the code that gets added to the final project.
+It does not include packages in a repositories that don't end up being compiled.
+
+However, it's still recommended to run linting and review the new repositories fully,
+because they may contain linter errors that help to find bugs.
```

Comment:
I believe this comment should be clearer. Instructions in `Dependencies.md` require linting and review of new repositories fully; do you mean that we should also review dependencies that don't end up being used?

Reply:
No, it's more of... you are running linting, then it's probably easier to run in on the whole module, rather than specific packages separately.

Reply:
I tried to clarify.